.prettierignore

```diff
@@ -14,3 +14,5 @@ node_modules/
 !/src/**/*.ts
 !/src/**/*.tsx
 !/src/**/*.js
+# The below causes a bunch of prettier warnings, but we might also want to be formatting these.
+# !/fixtures/**/*.ts
```

Comment:
Might be worth enabling this in a separate PR. It's always nice to have pretty things!

Reply:
Aye, I've kept it here partly because I didn't want to have to fight prettier again if the fixture needed updating. Once this is merged I'll do a PR to uncomment it and prettify everything

Reply:
And also because otherwise I feel like I'd have to squash the commit history down to explain away why it was added and then removed again.